cmd/manager/main.go

```diff
@@ -128,32 +125,37 @@ func main() {
 		Scheme:    mgr.GetScheme(),
 	}).SetupWithManager(mgr)
 	if err != nil {
-		log.Printf("unable to register instance controller to the manager: %v", err)
+		log.Printf("❌ unable to register instance controller to the manager: %v", err)
 		os.Exit(1)
 	}
+	log.Print("📙 Instance controller")
 
 	if strings.ToLower(os.Getenv("ENABLE_WEBHOOKS")) == "true" {
-		log.Printf("Setting up webhooks")
+		log.Printf("🔸 Setting up webhooks")
 
-		if err := registerWebhook("/validate", &v1beta1.Instance{}, &webhook.Admission{Handler: &kudohook.InstanceAdmission{}}, mgr); err != nil {
-			log.Printf("unable to create instance validation webhook: %v", err)
+		if err := registerWebhook("/admit", &v1beta1.Instance{}, &webhook.Admission{Handler: &kudohook.InstanceAdmission{}}, mgr); err != nil {
+			log.Printf("❌ unable to create instance admission webhook: %v", err)
 			os.Exit(1)
 		}
+		log.Printf("🧲 Instance admission webhook")
 
 		// Add more webhooks below using the above registerWebhook method
 	}
 
 	// Start the KUDO manager
-	log.Print("Starting KUDO manager")
+	log.Print("🏄 Done! Everything is setup, starting KUDO manager now")
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-		log.Printf("unable to run the manager: %v", err)
+		log.Printf("💀 unable to run the manager: %v", err)
 		os.Exit(1)
 	}
 }
 
 // registerWebhook method registers passed webhook using a give prefix (e.g. "/validate") and runtime object
 // (e.g. v1beta1.Instance) to generate a webhook path e.g. "/validate-kudo-dev-v1beta1-instances". Webhook
 // has to implement http.Handler interface (see v1beta1.InstanceAdmission for an example)
+//
+// NOTE: generated webhook path HAS to match the one, used in the webhook configuration. See for example how
+// MutatingWebhookConfiguration.Webhooks[0].ClientConfig.Service.Path is set in pkg/kudoctl/kudoinit/prereq/webhook.go:101
```

Comment:
AFAICT, that line is:
```
	deployment, err := client.KubeClient.AppsV1().Deployments("cert-manager").Get("cert-manager", metav1.GetOptions{})
```
so this comment has rotten even before it's been merged :see_no_evil: 
And I think what you refer to is on line 200.

Can you make this more robust, i.e. rely on code rather than comments, like use a common constant in both places or something?

Reply:
Good catch. Done